### DISCUSSION TITLE
Edge behaviour Custom Nodes

### DISCUSSION THREAD

**Discussion by MrVibe** [xyflow/xyflow#4609] [Q&A]
Facing a strange issue on how to mark the edge as output in custom node. What attributes are required to change this ? any example would be helpful.
<img width="264" alt="Screenshot 2024-08-31 at 4 33 12 PM" src="https://github.com/user-attachments/assets/ded601bb-ec93-4a4a-a06e-1d320e9f0d56">

**Comment by bcakmakoglu** [comment]
Can you provide us your custom node code?
Are you also using custom edges? If yes, what do they look like (the code)?

**Comment by MrVibe** [comment]
This was solved, the connect was defined on wrong edge.
